conf/deployment/fusion_hci_pc/client_bm_upi_1az_rhcos_nvme_3m_3w.yaml

```diff
@@ -0,0 +1,13 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetal'
```

Comment:
Should we use `baremetal` as platform? The deployment automation will differ from what we have for baremetal afaiu. Maybe we should introduce a new platform for fusion hci.

Reply:
The base platform/hardware where we can use to deploy this setup can be 

- vsphere 
- IBM BM instances
- baremetal

So for deployment we need either of above and that specific configuration then HCP/HCI specific configuration we need to update in above draft.  I am using the baremetal hardware on which this setup is deployed so just added this 2 configuration

Reply:
I still think that this attribute should be different because based on this is selected installation class and platform specific checks. This will be different from standard baremetal deployment but we can keep it for now and change it when needed.

Reply:
Filip has a point. 
If set to `baremetal` here it could cause certain conditions in ocs-ci to be met.
Here is an example https://github.com/red-hat-storage/ocs-ci/blob/1e93e43af7cb6c44760e9c720f58c28740317bcb/ocs_ci/ocs/platform_nodes.py#L84